### PR TITLE
Update getting_started.md

### PR DESCRIPTION
**Pull request by clamoriniere**
### What does this PR do?

Update the getting started doc.

### Motivation

current getting started is not up to date.

### Additional Notes

N/A

### Describe your test plan

try to deploy the agents by following the getting started doc.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -16,28 +16,37 @@ Using the Datadog Operator requires the following prerequisites:
...
+         name: "gcr.io/datadoghq/agent:latest"
+     clusterAgent:
+       image:
+         name: "gcr.io/datadoghq/cluster-agent:latest"
    ```
 
    Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your [Datadog API and application keys][4]
 
-3. Deploy the Datadog agent with the above configuration file:
+2. Deploy the Datadog agent with the above configuration file:
```
Comment (xornivore) [inline docs/getting_started.md:47]: Pretty sure you can keep this as 1. and have markdown update automatically when rendered.

**Review by xornivore — APPROVED** [review]

File: docs/getting_started.md (hunk 1 above)
Comment (clamoriniere) [reply docs/getting_started.md]: I haven't found in the github doc

Comment (xornivore) [reply docs/getting_started.md]: [check it out](https://gist.github.com/xornivore/3c4c915e8c85d26e269387475e6de596)

Comment (clamoriniere) [reply docs/getting_started.md]: PR updated

**Pull request merged**